include/wx/gtk/private/mediactrl.h

```diff
@@ -0,0 +1,46 @@
+#ifndef _WX_GTK_PRIVATE_MEDIACTRL_H_
```

Comment:
Trivial, but could you please add the usual `////...` header comment to the new file for consistency? Just copy it from another file (but use your own name, of course)?

Reply:
added